### PR TITLE
Persian(fa) Package

### PR DESCRIPTION
**Pull request by Ehsan-Home**
Hi.

In this pull request, I have added the Persian(fa) package to solid docs.

I have translated some parts of the `comparison.md` file and the translation of the rest is ongoing.

Please let me know if any files need to be changed.
Thanks 👋🏻

### CODE REVIEW

File: langs/fa/guides/comparison.md [hunk 1]
```diff
@@ -0,0 +1,33 @@
+# مقایسه با بقیه کتابخانه ها
+
+این بخش نمی تواند از برخی سوگیری ها دوری کند، اما ما فکر می کنیم مهم است که بدانیم راه حل Solid در مقایسه با کتابخانه های دیگر در کجا قرار دارد. این قسمت مربوط به عملکرد نیست و فقط تفاوت فلسفه های کتابخانه ها را بیان میکند. برای ارزیابی عملکرد، به
+[JS Framework Benchmark](https://github.com/krausest/js-framework-benchmark)
+نگاه کنید.
+
+## ری اکت (React)
```
Comment (aminya) [inline langs/fa/guides/comparison.md]: Could you use semi-space in the React name? (نیم فاصله)
```suggestion
## ری‌اکت (React)
```

File: langs/fa/guides/comparison.md [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+
+ری اکت (React) تأثیر زیادی روی Solid داشته است. سالید ایده (Solid) جریان یک طرفه ارسال داده ها و تفکیک صریح خواندن و نوشتن در Hooks ها را از React الهام گرفته است. Solid ایده ها و نظرات قوی و جذابی در مورد چگونگی رویکرد مدیریت داده در توسعه برنامه دارد، اما به دنبال محدود کردن اجرای آن نیست.
```
Comment (aminya) [inline langs/fa/guides/comparison.md]: Could you replace the remaining English words and names with their translation? Here I replaced some of them as an example. 

You can use this Wikipedia page to see the equivalent of some of the words:
https://fa.wikipedia.org/wiki/%D8%B1%DB%8C%E2%80%8C%D8%A7%DA%A9%D8%AA

```suggestion
ری‌اکت (React) تأثیر زیادی روی سالید (Solid) داشته است. سالید ایده جریان یک طرفه ارسال داده ها و تفکیک صریح خواندن و نوشتن در هوک ها (Hooks) را از ری‌اکت الهام گرفته است. سالید ایده ها و نظرات قوی و جذابی در مورد چگونگی رویکرد مدیریت داده در توسعه برنامه دارد، اما به دنبال محدود کردن اجرای آن نیست.

```

**Review by aminya — CHANGES_REQUESTED** [review]
Looks great. I left some suggestions. Thanks for initiating this! 💯

File: langs/fa/guides/comparison.md (hunk 2 above)
Comment (Ehsan-Home) [reply langs/fa/guides/comparison.md]: Hi Amin.

Thanks for the suggestions!

I added the Persian equivalents and technical english names in parentheses.

**Review by aminya — APPROVED** [review]

**Pull request merged**